app/client/src/modules/ui-builder/ui/wds/WDSCustomWidget/component/index.tsx

```
@@ -0,0 +1,237 @@
+              });
+            }
+
+            break;
+          case EVENTS.CUSTOM_WIDGET_UPDATE_MODEL:
+            props.update(message.data);
+            break;
+          case EVENTS.CUSTOM_WIDGET_TRIGGER_EVENT:
+            props.execute(message.data.eventName, message.data.contextObj);
+            break;
+          case EVENTS.CUSTOM_WIDGET_UPDATE_HEIGHT:
+            const height = message.data.height;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)